src/postgres/commands/connectPostgresDatabase.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureTreeItem, IActionContext } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { setConnectedDatabase } from "../setConnectedDatabase";
+import { PostgresDatabaseTreeItem } from "../tree/PostgresDatabaseTreeItem";
+import { connectedDBKey } from "./registerPostgresCommands";
+
+export async function connectPostgresDatabase(context: IActionContext, treeItem?: PostgresDatabaseTreeItem): Promise<void> {
+    if (!treeItem) {
+        treeItem = <PostgresDatabaseTreeItem>await ext.tree.showTreeItemPicker(PostgresDatabaseTreeItem.contextValue, context);
+    }
+
+    const oldTreeItemId: string | undefined = ext.connectedMongoDB && ext.connectedMongoDB.fullId;
```

Comment:
Did you forget to change `connectedMongoDB` to `connectedPostgresDB ` or do we not support connecting both at the same time?

Reply:
Yeah I should have changed that